### PR TITLE
Update pr-data.csv, add one ID test and fixed it

### PR DESCRIPTION
**Pull request by yijut2**
I found one ID flaky test in linkis repo under the links-commons module. The flaky test logs could be found in my VM `yijujt2@fa23-cs527-073.cs.illinois.edu`: `/home/yijujt2/linkis/linkis-id-detect.log`

The failure looks like:
```
[ERROR] Failures: 
[ERROR]   ExceptionManagerTest.testGenerateException:55 expected: <LinkisException{errCode=10000, desc='The map cannot be parsed normally, the map is empty or the LEVEL value is missing:(map不能被正常的解析，map为空或者缺少LEVEL值: ){desc=test, port=0, errCode=1, level=null, ip=null, serviceKind=null}', ip='null', port=0, serviceKind='null'}> but was: <LinkisException{errCode=10000, desc='The map cannot be parsed normally, the map is empty or the LEVEL value is missing:(map不能被正常的解析，map为空或者缺少LEVEL值: ){errCode=1, ip=null, desc=test, port=0, level=null, serviceKind=null}', ip='null', port=0, serviceKind='null'}>
[INFO] 
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

I also fixed this ID and opened the PR for it, the PR link is https://github.com/apache/linkis/pull/5005

### CODE REVIEW

File: format_checker/forked-projects.json [hunk 1]
```diff
@@ -463,4 +464,4 @@
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
   "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
-}
```
Comment (darko-marinov) [inline format_checker/forked-projects.json:466]: Please don't make these unnecessary changes.

File: pr-data.csv [hunk 2]
```diff
@@ -2506,6 +2506,7 @@ https://github.com/apache/karaf,b368a87cba0f0e235d5dae72fd6c3559b50fb75c,.,org.a
 https://github.com/apache/kylin,31ab93618311cde8b95ca400811861002d7eaefb,stream-coordinator,org.apache.kylin.stream.coordinator.assign.DefaultAssignerTest.reBalanceTest,ID,Accepted,https://github.com/apache/kylin/pull/1133,
 https://github.com/apache/kylin,31ab93618311cde8b95ca400811861002d7eaefb,stream-coordinator,org.apache.kylin.stream.coordinator.CoordinatorTest.testReassignFailOnStartNew,ID,Accepted,https://github.com/apache/kylin/pull/1119,
 https://github.com/apache/kylin,31ab93618311cde8b95ca400811861002d7eaefb,stream-coordinator,org.apache.kylin.stream.coordinator.CoordinatorTest.testReassignFailOnStopAndSync,ID,Accepted,https://github.com/apache/kylin/pull/1119,
+https://github.com/apache/linkis,2b47014684a0f8b2491fa5b90410e10537fcfaa6,linkis-commons,org.apache.linkis.common.exception.ExceptionManagerTest.testGenerateException,ID,Opened,https://github.com/apache/linkis/pull/5005,
```
Comment (darko-marinov) [inline pr-data.csv:2509]: Please change to `Accepted`.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Two changes.

File: pr-data.csv (hunk 2 above)
Comment (yijut2) [reply pr-data.csv:2509]: Updated, thanks.

File: format_checker/forked-projects.json (hunk 1 above)
Comment (yijut2) [reply format_checker/forked-projects.json:466]: I wonder how do I cancel the change?

Comment (darko-marinov) [reply format_checker/forked-projects.json:466]: @zzjas can help in class today if needed.
